compiler/gcc-cross-compile.cmake

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2015 Alexandre Pretyman
+# All rights reserved.
+#
+# ------------------------------------------------------------------------------
+#  GCC Based Cross Compiler CMake toolchain file
+#
+#  Variables:
+#   CROSS_COMPILE_TOOLCHAIN_PATH=/path/to/toolchain/bin
+#   CROSS_COMPILE_TOOLCHAIN_PREFIX=arm-unknown-linux-gnueabihf
+#   CROSS_COMPILE_SYSROOT=/path/to/sysroot [optional]
+# ------------------------------------------------------------------------------
+
+if(DEFINED POLLY_COMPILER_GCC_CROSS_COMPILE)
+  return()
+else()
+  set(POLLY_COMPILER_GCC_CROSS_COMPILE TRUE)
+endif()
+
+if( CMAKE_TOOLCHAIN_FILE )
+  # touch toolchain variable to suppress "unused variable" warning
+endif()
+
+string(COMPARE EQUAL
+    "${CROSS_COMPILE_TOOLCHAIN_PATH}"
+    ""
+    _is_empty
+)
+if(_is_empty)
+  polly_fatal_error("CROSS_COMPILE_TOOLCHAIN_PATH not set. Set it to the absolute path of the \"bin\" directory for the toolchain")
```

Comment:
I guess we should check `CROSS_COMPILE_TOOLCHAIN_PATH` CMake variable in `compiler/gcc-cross-compile.cmake` and report `CROSS_COMPILE_TOOLCHAIN_PATH not set` (internal error) and check `RASPBERRYPI_CROSS_COMPILE_TOOLCHAIN_PATH` environment variable in `raspberrypi2-cxx11.cmake` and report `RASPBERRYPI_* environment variable not set, please set it to ...` (user error).